datafusion/sqllogictest/test_files/unnest.slt

```diff
@@ -36,7 +36,7 @@ select unnest([1,2,3]);
 2
 3
 
-query error DataFusion error: Error during planning: unnest\(\) can only be applied to array and structs and null
+query error DataFusion error: Error during planning: unnest\(\) can only be applied to array and struct
 select unnest(null);
```

Comment:
This test shows that `null` is not supported, which conflicts with the error message.

Reply:
It is an error because unnest with null is not supported, but it should, so the error message contains null and struct.

Reply:
Maybe we can checking null beforehand and return not yet implemented error, so does struct

Reply:
Can you also add null checking and return not yet implemented error